GitUI/CommandsDialogs/UserScriptContextMenuExtensions.cs

```diff
@@ -45,7 +46,8 @@ public static void AddUserScripts(this ContextMenuStrip contextMenu, ToolStripMe
                 item.Click += (s, e) =>
                 {
                     int scriptId = script.HotkeyCommandIdentifier;
-                    scriptInvoker(scriptId);
+                    var scriptsRunner = hostControl?.UICommands?.GetRequiredService<IScriptsRunner>();
+                    scriptsRunner?.RunScript(scriptId, hostControl?.FindForm() as GitModuleForm, hostControl);
```

Comment:
I don't see how this is supposed to work. It would fail on `hostControl.FindForm()`, because `hostControl` was removed above. And `IScriptHostControl` doesn't have a `FindForm`, so it can't act as a replacement.

Reply:
You are right. I did not notice the call `hostControl.FindForm()`.
So just remove the pointless `.?`.

Reply:
Already removed the unnecessary `?`s in that function.